virt/virt-learn-more-about-openshift-virtualization.adoc

```diff
@@ -54,23 +57,28 @@ Configure additional networks::
 ** Configure an xref:../virt/virtual_machines/vm_networking/virt-configuring-sriov-device-for-vms.adoc#virt-configuring-sriov-device-for-vms[SR-IOV network device].
 ** xref:../virt/virtual_machines/vm_networking/virt-attaching-vm-to-sriov-network.adoc#virt-attaching-vm-to-sriov-network[Attach a virtual machine] to the SR-IOV network.
 
-Monitor your {VirtProductName} environment::
-* Monitor resources, details, status, and top consumers on the  xref:../virt/logging_events_monitoring/virt-reviewing-virtualization-overview.adoc#virt-reviewing-virtualization-overview[*Virtualization Overview* page].
+Monitoring your {VirtProductName} environment::
+* Monitor resources, details, status, and top consumers on the  xref:../virt/logging_events_monitoring/virt-reviewing-virtualization-overview.adoc#virt-reviewing-virtualization-overview[*Overview* page].
 * View high-level information about your virtual machines on the xref:../virt/logging_events_monitoring/virt-viewing-information-about-vm-workloads.adoc#virt-viewing-information-about-vm-workloads[*Virtual Machines* dashboard].
 * View virtual machine xref:../virt/logging_events_monitoring/virt-logs.adoc#virt-logs[logs].
 
-Automate your deployments::
+Automating deployments::
 * xref:../virt/virtual_machines/advanced_vm_management/virt-automating-management-tasks.adoc#virt-automating-virtual-machine-creation-with-ansible_virt-automating-management-tasks[Automate virtual machine deployments] with Ansible.
 * xref:../virt/virtual_machines/virt-automating-windows-sysprep.adoc#virt-automating-windows-sysprep[Automate Windows virtual machine deployments] with `sysprep`.
 
 [id="{context}_additional-resources"]
 [role="_additional-resources"]
 == Additional resources
 
+Before you begin::
+* xref:../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[Using a CSI-enabled storage provider]
+* xref:../virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc#virt-configuring-local-storage-for-vms[Configuring local storage]
 * xref:../networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator.adoc#k8s-nmstate-about-the-k8s-nmstate-operator[About the Kubernetes NMState Operator]
 * xref:../virt/virtual_machines/advanced_vm_management/virt-specifying-nodes-for-vms.adoc#virt-specifying-nodes-for-vms[Specifying nodes for virtual machines]
-* xref:../virt/live_migration/virt-live-migration.adoc#virt-live-migration[Live migration]
+
+Getting started::
 * xref:../virt/vm_templates/virt-creating-vm-template.adoc#virt-creating-vm-template[Virtual machine templates]
-* xref:../virt/virtual_machines/virtual_disks/virt-configuring-local-storage-for-vms.adoc#virt-configuring-local-storage-for-vms[Configuring local storage]
 
+Next steps::
+* xref:../virt/live_migration/virt-live-migration.adoc#virt-live-migration[Live migration]
```

Comment:
I recall that there was an issue with Jupiter and using any formatting other than unordered lists in an additional resources section. [The contribution guide still seems to indicate this.](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections) For this page, it might make sense to add the links to the existing sections, maybe as sub-bullets to differentiate them from the more basic resources?

Reply:
Sure, that works for me!